core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/DhcCalendarController.kt

```diff
@@ -0,0 +1,30 @@
+package com.dhc.designsystem.calendar
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import java.time.LocalDate
+
+@Stable
+class DhcCalendarController(
+    private val initialDate: LocalDate = LocalDate.now(),
+    private val initialPage: Int = Int.MAX_VALUE / 2,
+) {
+    val currentDate = mutableStateOf(initialDate)
+
+    fun getDate(page: Int) = initialDate.plusDays(page.toLong() - initialPage)
+
+    fun onChangePage(page: Int) {
+        currentDate.value = initialDate.plusDays(page.toLong()- initialPage)
```

Comment:
```suggestion
        currentDate.value = getDate(page)
```
Nit : 이미 구현된 함수를 활용해보면 좀 더 간단하게 표현할 수 있을 것 같아용

Reply:
@JeonK1 
와 그러네?? ㅋㅋ
날카롭다 미쳣다